internal/hcl/block.go

```diff
@@ -843,6 +853,11 @@ func (b *Block) GetAttributes() []*Attribute {
 
 	hclAttributes := b.getHCLAttributes()
 
+	// Remove the depends_on meta attribute since it never needs to be evaluated
+	if b.Type() == "resource" || b.Type() == "data" || b.Type() == "module" {
+		delete(hclAttributes, "depends_on")
+	}
```

Comment:
can we move this into the actual `getHCLAttributes` method instead of here, this means we don't have to allocate redundant data to the `hcl.Attributes` map.